src/main/java/mate/academy/Main.java

```diff
@@ -27,7 +33,8 @@
         secondCinemaHall.setCapacity(200);
         secondCinemaHall.setDescription("second hall with capacity 200");
 
-        CinemaHallService cinemaHallService = null;
+        CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
```

Comment:
Similar to the previous comment, ensure that the `injector.getInstance(CinemaHallService.class)` returns an instance of `CinemaHallService` to avoid `ClassCastException`.